fastlane/README.md

```
@@ -0,0 +1,104 @@
+
+## Android
+
+### android test
+
+```sh
+[bundle exec] fastlane android test
+```
+
+Runs all the tests
+
+### android distribute_staging_debug
```

Markdownlint (0.37.0)

Expected: setext; Actual: atx
Heading style

(MD003, heading-style)